testing/data/spv/binary/valid_1000_nested.hex

```diff
@@ -0,0 +1 @@
1adb9244121030c064fa8459d77ad1b4da374b30b02d7ed8422175bde8d0968b96522acfc8bf0ffffffff02e8030000000000001976a914bdb1b777c7017f08264d04843bc64f32461d654c88ac15070000000000001976a914bdb1b777c7017f08264d04843bc64f32461d654c88ac0000000001fd760101000000028fdb2d3634197d89e8e8c4e3205d7f5add0c036aff58be88fb9586749c9741cd000000006b48304502210083d4acc3e5c46b82d4d276699ac88efa794def7cd2d017352faf199b19786f0d022025edf68bf029b0b1b9d20bbc7119babc4e006d98db878f8a865ace97982228dd4121030c064fa8459d77ad1b4da374b30b02d7ed8422175bde8d0968b96522acfc8bf0ffffffffc2aaf7634bd6b441c384b90a93b19c6bcab4a9ddda5afb6be1b49e542d8d03998f0200006b4830450221009bd82cb3984d36b24f9dc929531d1e4d307e5157f7873b73b5bedaa9a2cc9b7c02206f3bc5207b2b33871b009af18972c608385715f01133e5a6767727f9899edde84121030c064fa8459d77ad1b4da374b30b02d7ed8422175bde8d0968b96522acfc8bf0ffffffff02e8030000000000001976a914bdb1b777c7017f08264d04843bc64f32461d654c88ac15070000000000001976a914bdb1b777c7017f08264d04843bc64f32461d654c88ac0000000001fd7601010000000223c3269edb0f55d00e542ce072e09a6162fea00d719a4be410d66d28fc3628e0000000006b483045022100ccc159941a5f831eea30b45ed807942151cf5f38cb7870f66430c3fd7784b6d802205adede17cedad787ee199c07383a0fa92b91da33cc2e1838ce73be7d1f01ea124121030c064fa8459d77ad1b4da374b30b02d7ed8422175bde8d0968b96522acfc8bf0ffffffffc2aaf7634bd6b441c384b90a93b19c6bcab4a9ddda5afb6be1b49e542d8d03998e0200006b483045022100af5183486c832ce7c4506dae6fe91ce8d06ff8f26cc67486198a8135504cb42c02202d2fcaf85f502cbcb7c17914cc82d738a7c093a9be8573a1d27d0dcfa5e773b34121030c064fa8459d77ad1b4da374b30b02d7ed8422175bde8d0968b96522acfc8bf0ffffffff02e8030000000000001976a914bdb1b777c7017f08264d04843bc64f32461d654c88ac15070000000000001976a914bdb1b777c7017f08264d04843bc64f32461d654c88ac0000000001fd75010100000002caf4f066f09ae27c6d24f0b2a6e1c60330b6da37327245d7b06930dbf4d90246000000006b483045022100e9caa53ff7c1a9bcfbed9a1898a942e1f6d9600beee80dc4eac80ddd7e0ae109022037af3a1b372069f8d8f0bd0e0a234d5e644ad5bed3420e87b1c090029f0d2a1e4121030c064fa8459d77ad1b4da374b30b02d7ed8422175bde8d0968b96522acfc8bf0ffffffffc2aaf7634bd6b441c384b90a93b19c6bcab4a9ddda5afb6be1b49e542d8d03998d0200006a4730440220137986a1f486d38b727240c568603996e6d76d940444fa7491eb20eb44f72c2a022062cca29c4d7505a6e8eab763b9050f2f1b8ff0be71eb84933b75a76f682665ef4121030c064fa8459d77ad1b4da374b30b02d7ed8422175bde8d0968b96522acfc8bf0ffffffff02e8030000000000001976a914bdb1b777c7017f08264d04843bc64f32461d654c88ac15070000000000001976a914bdb1b777c7017f08264d04843bc64f32461d654c88ac0000000001fd76010100000002444ba809d9ba0ffef870591addf45e3852c4063dd8eb736b8b0754d095b8cade000000006b483045022100c99049adf2f3f9923135a5775f8f1c2f498ad66464957d0ba966f46ceb82f20a02206808ccfe7427658ecc817cd3ef356ccf377acea3481d0e2569a103abc037cb834121030c064fa8459d77ad1b4da374b30b02d7ed8422175bde8d0968b96522acfc8bf0ffffffffc2aaf7634bd6b441c384b90a93b19c6bcab4a9ddda5afb6be1b49e542d8d03998c0200006b483045022100a6c8cb3397841e7e7fb954a70410f496992ecbbbd20641506f8bac3a93748e5b02200e919bd97f950f5f18ff60a4eef4221f934
```

Comment:
If this is an absolutely massive tx + ancestry, is there any chance you could print it out as JSON and add it to the json test data, and maybe add a single "passes" test case for it too?
I'm thinking it might be good to have a clear text representation of an absolutely gigantic data struct, what do you think @jadwahab

Reply:
yupp good idea

Reply:
Happy to try, I think it might be too many nested layers for JSON. Hmm looking it up different implementations in different languages have different cutoffs. Olden days systems were 32 levels, PHP is 512, default in some modern libraries is 10,000 and some people up it to 30,000 nested levels.

Reply:
The idea with including these binary examples was specifically to test extremely large levels of nesting which are not readable.